models/db.go

```diff
@@ -0,0 +1,49 @@
+package models
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "postgres"
+	password = "vagrant"
+	dbname   = "patches"
+  )
+
+  type Datastore interface {
+	CreatePatch(patch *Patch) (error)
+	GetPatches(filterString string) ([]Patch, error)
+	DeletePatches(convo_id int64) (int64, error)
+}
+
+// DB represents an SQL database connection
+type DB struct {
+	*sql.DB
+}
+
+// Connect to database
+func DBConnect() (*DB, error) {
```

Comment:
This function could just take in the `psqlInfo` as an argument and then the calling code can just build the string itself, using the environment variables like I mentioned above.